From a87cce394269985c1dad0510678d8be8dd8eefb2 Mon Sep 17 00:00:00 2001
From: Adrian Schmutzler <freifunk@adrianschmutzler.de>
Date: Wed, 9 May 2018 16:04:04 +0200
Subject: [PATCH] Revert "babeld: release 1.8.1"

This reverts commit 7fc2fbdfc1b79953360b8f535c1cb9106b20fe3e.

Babel 1.8.1 breaks ipv4 announcements ...

Signed-off-by: Adrian Schmutzler <freifunk@adrianschmutzler.de>
---
 babeld/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/babeld/Makefile b/babeld/Makefile
index db2b32c..cd7f6e3 100644
--- a/babeld/Makefile
+++ b/babeld/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=babeld
-PKG_VERSION:=1.8.1
-PKG_RELEASE:=1
+PKG_VERSION:=1.8.0
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://www.irif.fr/~jch/software/files/
-PKG_MD5SUM:=a57caa5be996c61bd6a1616fdc01d807
+PKG_MD5SUM:=eb1c66c382e9181c418ebd84e52b5af2
 PKG_LICENSE:=MIT
 
 include $(INCLUDE_DIR)/package.mk
-- 
2.7.4

